packages/stylelint-plugin/test/no-prefix-literal.test.js

```diff
@@ -0,0 +1,57 @@
+/* Copyright 2020 Palantir Technologies, Inc. All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.*/
+
+// eslint-disable-next-line import/no-extraneous-dependencies
+const { expect } = require("chai");
+const stylelint = require("stylelint");
+
+const config = {
+    plugins: ["@blueprintjs/stylelint-plugin"],
+    rules: {
+        "@blueprintjs/no-prefix-literal": true,
+        "at-rule-semicolon-newline-after": "always",
+    },
+};
+
+it("Warns when .bp3 is present", async () => {
```

Comment:
is there a way to test the autofix portion?

Reply:
could we also test css module syntax? e.g.

```
.my-class {
  :global(.bp3-card) {
  }
}
```

should trigger as well

Reply:
I spent an hour trying to figure out how to test the autofixer and I couldn't figure it out. The problem is that when `fix: true` is passed to stylelint then it modifies the original file which we don't want. If someone finds a working example online I'd love to test it, but otherwise I could try adding tests for some of the functions that I'm using to perform autofixing (like inserting imports and etc)

Reply:
here's one example i found: https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-alphabetical-order/tests/index.js

i care more about catching the css module syntax than the fixer testing if that helps (unit testing the individual functions sound ok to me too)

Reply:
This uses jest which blueprint unfortunately doesn't use. I added some tests for the util functions that we use to do autofixing

Reply:
I'm open to adding Jest in a future PR for certain BP packages if it helps us leverage more things in the ecosystem